test/models/person_model.rb

```diff
@@ -1,11 +1,19 @@
 require 'active_model'
 
 class PersonModel
-  include ActiveModel::Model
+  extend  ActiveModel::Naming
+  extend  ActiveModel::Translation
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
   include GlobalID::Identification
+  include ActiveModel::AttributeMethods
 
   attr_accessor :id
 
+  def initialize(options={})
```

Comment:
There's surely a more clever way of doing this with ActiveModel modules, but is it worth it?